### ISSUE TITLE
Inlined SVG's need a `<title>` tag in order to be accessible.

### ISSUE THREAD

**Issue by trackleft** [az-digital/az_quickstart#498]
## Problem/Motivation
Inlined SVGs do not follow the normal `<img src="" alt="" />` pattern and those need to be handled a different way when considering image accessibility.

### Describe the bug
No alt text on inlined SVG

### To Reproduce
In Appearance > Themes > AZ Barrio Settings find the inline svg option under logo and footer logo, and enable them, upload an SVG that doesn't have a `<title>` element and check if a screen reader can discern what the image is.

https://css-tricks.com/accessible-svgs/#2-inline-svg
